04_verify.sh

```diff
@@ -18,6 +18,11 @@ source lib/images.sh
 if [ "${EPHEMERAL_CLUSTER}" == "tilt" ]; then
   exit 0
 fi
+# TODO (mboukhalfa) Skip verification related to virsh
+if [[ "${NODES_PLATFORM}" == "fake" ]]; then
+  echo "Skipping virsh nodes verification on fake vm platform"
+  exit 0
```

Comment:
Can we move this down to line ~260? Then we will still verify that the kubernetes cluster is there and has some resources.

Reply:
I moved it to 240 because with the fake we do not apply BMH we do need to verify them